be/src/Project/Services/Inventory/Inventory.Api/Services/InventoryItems/InventoryService.cs

```diff
@@ -475,5 +476,27 @@ public async Task UpdateInventoryQuantity(Guid? productId, double bookedQuantity
                 await _context.SaveChangesAsync();
             }
         }
+
+        public IEnumerable<InventorySuggestionServiceModel> GetInventorySuggestions(GetInventorySuggestionsServiceModel model)
```

Comment:
**suggestion:** I would simplify this method like this

![obraz](https://github.com/user-attachments/assets/bd5d34e1-44f4-4ace-b32a-709ea2f138ed)